Add refresh token documentation specific to the OIE SDK and widget (#2291)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-refresh-tokens/main/aspnet/refreshusingthesdk.md

## Option 1: Refresh using the .Net SDK

> The [.Net SDK, Widget and sample app](https://github.com/okta/okta-idx-dotnet)

Reviewer: ```suggestion
The [.Net SDK, Widget, and sample app](https://github.com/okta/okta-idx-dotnet)
```

Author: See above comment about commas.

Author: Will add the comma.

> do not currently have built-in support for refreshing tokens. Instead use

Reviewer: ```suggestion
don't currently have built-in support for refreshing tokens. Instead use
```
